src/dex-ui/layouts/DashboardLayout.tsx

```diff
@@ -0,0 +1,178 @@
+import { Box, Center, Flex, Tab, TabList, Tabs } from "@chakra-ui/react";
+import { ErrorLayout, LoadingSpinnerLayout, NotFound, Page, PageLayout } from "@layouts";
+import {
+  Color,
+  Notification,
+  NotficationTypes,
+  useNotification,
+  LayoutIcon,
+  TransactionIcon,
+  BoxIcon,
+  LockIcon2,
+  UsersIcon,
+  SettingsIcon,
+} from "@dex-ui-components";
+import { useTabFilters } from "@hooks";
+import { useLocation, NavLink, useNavigate, useParams } from "react-router-dom";
+import { PropsWithChildren } from "react";
+import { DAO } from "@services";
+import { Paths } from "@dex-ui/routes";
+import { DashboardHeader } from "@dex-ui/components/DashboardHeader";
+
+const daoNavigationTabs = [
+  {
+    icon: <LayoutIcon boxSize="5" />,
+    title: "Dashboard",
+  },
+  {
+    icon: <TransactionIcon boxSize="5" />,
+    title: "Transactions",
+  },
+  {
+    icon: <BoxIcon boxSize="5" />,
+    title: "Assets",
+  },
+  {
+    icon: <LockIcon2 boxSize="5" />,
+    title: "Staking",
+  },
+  {
+    icon: <UsersIcon boxSize="5" />,
+    title: "Members",
+  },
+  {
+    icon: <SettingsIcon boxSize="5" />,
+    title: "Settings",
+  },
+];
+
+interface DashboardLayoutProps extends PropsWithChildren {
+  dao?: DAO;
+  isNotFound: boolean;
+  isDAOFound: boolean;
+  isError: boolean;
+  isLoading: boolean;
+  errorMessage?: string;
+  isSuccess: boolean;
+}
+
+export function DashboardLayout(props: DashboardLayoutProps) {
```

Comment:
Layout components should only contain logic that arranges elements passed in as props/children. It should not contain any business logic. We should move this to the `pages/dao` folder and rename it `DAODashboard`.

Reply:
Moved to pages/dao.